changelog/2024-11-18T14_59_34+01_00_fix2845

```diff
@@ -0,0 +1 @@
+FIXED: Clash generates illegal Verilog names
```

Comment:
Could you mention the issue here? That way it will get picked up by https://github.com/clash-lang/clash-compiler/blob/master/changelog/comment-gh.py.

Reply:
Ah, will do.